### PR TITLE
Realistic Eerie biome

### PR DESCRIPTION
**Pull request by Zeno410**
Created realistic version of Eerie biome. Did not add it to the RTG layout - weight is set to 0 - it will only show up in CC layouts at present.

### CODE REVIEW

File: src/main/java/rtg/config/thaumcraft/ConfigTC.java [hunk 1]
```diff
@@ -41,12 +44,14 @@ public static void init(File configFile)
 			
 			generateTCTaintedLand = config.getBoolean(formatConfig("generateTCTaintedLand"), "Biomes", generateTCTaintedLand, "");
 			generateTCMagicalForest = config.getBoolean(formatConfig("generateTCMagicalForest"), "Biomes", generateTCMagicalForest, "");
+			generateTCMagicalForest = config.getBoolean(formatConfig("generateTCEerie"), "Biomes", generateTCEerie, "");
```
Comment (whichonespink44) [inline src/main/java/rtg/config/thaumcraft/ConfigTC.java]: should be `generateTCEerie =`

File: src/main/java/rtg/config/thaumcraft/ConfigTC.java [hunk 2]
```diff
@@ -41,12 +44,14 @@ public static void init(File configFile)
...
 			
 			weightTCTaintedLand = config.getInt(formatConfig("weightTCTaintedLand"), "Weights", weightTCTaintedLand, biomeWeightMin, biomeWeightMax, "");
 			weightTCMagicalForest = config.getInt(formatConfig("weightTCMagicalForest"), "Weights", weightTCMagicalForest, biomeWeightMin, biomeWeightMax, "");
 			
             villageTCTaintedLand = config.getBoolean(formatConfig("villageTCTaintedLand"), "Villages", villageTCTaintedLand, "");
             villageTCMagicalForest = config.getBoolean(formatConfig("villageTCMagicalForest"), "Villages", villageTCMagicalForest, "");
+            villageTCEerie = config.getBoolean(formatConfig("villageTCMagicalForest"), "Villages", villageTCEerie, "");
```
Comment (whichonespink44) [inline src/main/java/rtg/config/thaumcraft/ConfigTC.java]: should be `formatConfig("villageTCEerie")`

File: src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+import net.minecraft.world.biome.BiomeGenBase;
+
+public class RealisticBiomeTCEerie extends RealisticBiomeTCBase
+{
+	public RealisticBiomeTCEerie(BiomeGenBase tcBiome)
+	{
+		super(
+			tcBiome, BiomeBase.climatizedBiome(BiomeGenBase.river, Climate.TEMPERATE),
+			new TerrainTCMagicalForest(),
```
Comment (whichonespink44) [inline src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java]: Need to create TerrainTCEerie.java

File: src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+			new SurfaceTCMagicalForest(tcBiome.topBlock, tcBiome.fillerBlock)
```
Comment (whichonespink44) [inline src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java]: Need to create SurfaceTCEerie.java

File: src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+		);
+
+		this.setRealisticBiomeName("Thaumcraft Eerie");
+		this.biomeSize = BiomeSize.NORMAL;
+		this.biomeWeight = 0;
+		this.generateVillages = ConfigTC.villageTCMagicalForest;
```
Comment (whichonespink44) [inline src/main/java/rtg/world/biome/realistic/thaumcraft/RealisticBiomeTCEerie.java]: this.generateVillages = ConfigTC.villageTCEerie;

**Comment by whichonespink44** [comment]
Added a few comments - just a few instances of magical forest not replaced with eerie. Submit another pull request with the changes and I'll merge it.

**Comment by whichonespink44** [comment]
Actually, I'm adding biomes now, so I'll do this one as well.

**Comment by Zeno410** [comment]
There's no need for a Eerie surface or Terrain - as it currently generates in CC it's the same a Magical Forest except for the various Eerie effecst (more or less a dark version of Magical Forest). So you can just re-use for now

**Pull request closed without merging**